_episodes/07-branches.md

```diff
@@ -6,8 +6,10 @@ questions:
 - "What are branches?"
 - "How can I work in parallel using branches?"
 objectives:
-- "Understand why branches are useful for working on separate tasks in the same repository."
-- "Use branches for working in parallel"
+- "Understand why branches are useful for:"
+- "working on separate tasks in the same repository concurrently"
```

Comment:
Like these additions.  Having the list under the "Understand why branches are useful for:" one might make them format a little weirdly on the rendered version. You might want to add spaces and an extra '-' before like ...
"    - working on separate tasks in the same repository concurrently" which *I think* should kind of indent them.